doc/node-types-html/js/page/ExamplesView.js

```diff
@@ -1,199 +1,200 @@
-            });
-
-            /* loading selected image */
-            var imageURL = $(_thumb).data('large');
-
-           /* SVG Loading happens here */
-            var image = $("<img></img");
-            image.attr("src",imageURL);
-            image.attr('id','example-image');
-            image.data('code',$(_thumb).data('code'));
-
-            /* adding the loaded image into the drag view */
-            $("#example-image-container").append(image);
-
-            /* marking the active thumb */
-            $(_thumb).addClass('example-thumb-active');
-
-            /* making the image draggable */
-            image.draggable();
-
-            /* removing existing copy feedback and restoring the copy button */
-            if($('#example-info').has('#example-code-copied')){
-                $('#example-code-copied').remove();
-                $('#example-get-code').show();
-            }
-        }
-
-        /* --------------------------------------------------------
-         * COPY EXAMPLE CODE
-         * Implement the algorith to read the example
-         * code here. The var code already stores any key, url or
-         * string you sent when creating the Entry in the main loop
-         * --------------------------------------------------------
-         */
-
-        function copyExampleCode(event){
-
-            var code = $('#example-image').data('code');
-            $(this).fadeOut(100,showCopiedFeedback);
-        }
-        function showCopiedFeedback(){
-
-            var feedback = $('<img></img>');
-            feedback.attr('src','images/button_code_copied.png');
-            feedback.attr('id','example-code-copied');
-            $('#example-info').append(feedback);
-
-            feedback.hide();
-            feedback.fadeIn(100);
-        }
-
-        /* clears all data and loaded images
-         * closes the view and restore the main content */
-        function close(){
-            el.fadeOut(250,onCloseComplete);
-        }
-        function onCloseComplete(){
-            /* clearing all created objects */
-            $(window).off('resize',onWindowResize);
-            $("img[id^=thumb_]").each(function(){
-                $(this).remove();
-            });
-
-            $('#example-image-container').empty();
-            $('#example-get-code').remove();
-            $('#example h1, p').remove();
-            $('#example-code-copied').remove();
-
-            $('#example').hide();
-            el.trigger('examples:closed');
-        }
-        ExamplesView.prototype.show = function(_data){
-            /* showing the example view */
-            $("#example").fadeIn();
-
-            adjustExampleView();
-            loadExampleContents(_data);
-            $(window).on('resize',onWindowResize);
-        }
-    }
-
-    window.ExamplesView = ExamplesView;
-
-}(window));
+//Commenting out as modified version of this script is loaded with the page inline.
```

Comment:
@kamidipreetham  Also when such kind of change is required (moving code for another file) it's highly indicated to make a commit moving the file without any changes, and another commit just making changes on code itself, otherwise it's really hard to review it.